applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_linear_3D2N.cpp

```diff
@@ -50,7 +50,7 @@ void CrBeamElementLinear3D2N::CalculateLocalSystem(
   this->CalculateLeftHandSide(rLeftHandSideMatrix, rCurrentProcessInfo);
 
   Vector nodal_deformation = ZeroVector(msElementSize);
-  CrBeamElement3D2N::GetValuesVector(nodal_deformation); // modified by M.Fusseder
+  CrBeamElement3D2N::GetValuesVector(nodal_deformation); 

```

Comment:
would

        this->GetValuesVector 

work?

Reply:
Unfortunately not. I tried it but it doesn't work.